src/components/Auth/SignUp/SignUp.js

```diff
@@ -201,7 +189,40 @@ export default class SignUp extends Component {
   handleLogin = () => this.props.updateAuthDialog('login');
 
   render() {
-    const { fieldStyle, fontStyle, underlineFocusStyle } = styles;
+    const emailStyle = {
```

Comment:
Don't write CSS of individual elements in separate objects. Rather, make a `styles` object at the end of the file and add CSS of every element in there. See how it's done in other components.